docs/api/index.md

```diff
@@ -166,24 +166,74 @@ These global functions allows you to hook into the life cycle of tests to avoid
 
 Register a callback to be called before each test in the current context.
 
-**Type:** `beforeEach(fn: () => Awaitable<void>, timeout: number)`
+**Type:** `beforeEach(fn: () => Awaitable<void>, timeout?: number)`
+
+```ts
+import { beforeEach } from 'vitest'
+
+let count = 0 // total number of tests.
```

Comment:
We should remove the dots in all. Maybe the beforeEach example could be reworked to show something different.

Reply:
@patak-dev okay. I'll update that now

Reply:
@patak-dev checkout the new beforeEach example